src/main/java/pl/wedding/management/converter/Converter.java

```diff
@@ -0,0 +1,33 @@
+package pl.wedding.management.converter;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class Converter<T, U> {
```

Comment:
Jeżeli tworzymy klasy generyczne to fajnie, jeśli w Java Docu byśmy napisali czego tak naprawdę oczekujemy. Albo po prostu nazywajmy lepiej generyki. Czym jest T? Czym jest U? Nic nie stoi na przeszkodzie, żeby nazwać to DTO i ENTITY :) W innym wypadku otwierając taką klasę muszę spędzić trochę czasu na tym, żeby się domyślić co autor miał na myśli